src/search/SearchMessagesScreen.js

```diff
@@ -37,17 +38,17 @@ class SearchMessagesScreen extends PureComponent<Props, State> {
    *  Performs a network request associated with a query. Does not
    *  update or access internal state (except `auth`).
    */
-  performQueryRaw = async (query: string): Promise<Message[]> => {
+  performQueryRaw = async (query: string) => {
     const { auth } = this.props;
-    const { messages } = await api.getMessages(
+    const response = await api.getMessages(
       auth,
       SEARCH_NARROW(query),
       LAST_MESSAGE_ANCHOR,
       20,
       0,
       false,
     );
```

Comment:
Following from my comment at messageFetchComplete, please also do a similar refactor for `api.getMessages` (the default export of src/api/messages/getMessages.js), in its own commit. 🙂 In that case, though, `auth` can be left as the first param because it's kind of an ambient piece of data that's not specific to any API request, and the rest can be brought together into a single object parameter.